engine/security/src/main/scala/pl/touk/nussknacker/ui/security/oauth2/OpenIdConnectProfile.scala

```diff
@@ -0,0 +1,56 @@
+package pl.touk.nussknacker.ui.security.oauth2
+
+import io.circe.generic.JsonCodec
+import io.circe.generic.extras.{Configuration, ConfiguredJsonCodec, JsonKey}
+import pl.touk.nussknacker.ui.security.api.{LoggedUser, RulesSet}
+import pl.touk.nussknacker.ui.security.oauth2.OAuth2Profile.getUserRoles
+
+import scala.concurrent.duration.Deadline
+
+@ConfiguredJsonCodec case class OpenIdConnectUserInfo
+(
+  @JsonKey("sub") subject: Some[String],
```

Comment:
Either Option[String] or just String? :)

Reply:
This class implements the `JwtStandardClaim` interface. In a JWT the field is optional, but in the context of OIDC, it becomes mandatory.

Reply:
ok, I understand, nice idea :+1: though a comment why it's like that would be useful...

Reply:
I have added a comment.